mtgjson5/compiled_classes/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 from .mtgjson_all_identifiers import MtgjsonAllIdentifiersObject
 from .mtgjson_all_printings import MtgjsonAllPrintingsObject
+from .mtgjson_all_tcg_skus import MtgjsonAllTcgSkusObject
```

Comment:
would rename to MtgjsonAllTcgplayerSkusObject (and renamed down the line accordingly)